### PR TITLE
run imgops in docker

### PR DESCRIPTION
**Pull request by akash1810**
Imgops is the image resizer service. Currently you have to manually start it, which is easy to forget. Its a simple nginx server, so lets just run it in docker instead to simplfy the dev process.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,4 +10,5 @@ data
 .DS_Store
 *.py[cod]
 .venv
-kahuna/npm-debug.log
\ No newline at end of file
+kahuna/npm-debug.log
+imgops/dev/nginx.conf
```
Comment (akash1810) [inline .gitignore]: this file is generated in `dev-start.sh`

File: get-stack-resource.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (akash1810) [inline get-stack-resource.sh]: this is actually a `git mv` plus some changes... not sure why GH thinks its a new file...

File: imgops/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Img Opts for the Grid
+# ImgOps for the Grid
 Local version of the imgops service
 
 ## Requirements
```
Comment (shaundillon) [inline imgops/README.md]: We should add Docker to the requirements

Comment (akash1810) [reply imgops/README.md]: https://github.com/guardian/grid/blob/master/dev-start.sh#L27 😄

Comment (akash1810) [reply imgops/README.md]: updated the setup docs now too.

File: get-stack-resource.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    echo "Please specify a stack resource."
+    exit 1
+fi
+
+if [ -z ${STACK_NAME} ];
+then
+    STACK_NAME=media-service-DEV
+fi
+
+JQ_FILTER="jq '.StackResources[] | select(.LogicalResourceId == \"$STACK_RESOURCE\") | .PhysicalResourceId'"
```
Comment (LATaylor-guardian) [inline get-stack-resource.sh]: Does this script check that `jq` is installed and if not install it?

**Pull request merged**